routes/index.js

```diff
@@ -35,8 +35,8 @@ exports.index = function (req, res, next) {
 
 
 exports.admin = function (req, res, next) {
-  console.log(req.body);
-  User.find({ username: req.body.username, password: req.body.password }, function (err, users) {
+  const body = req.body;
```

Comment:
Your organization's best practices recommends the usage of destructuring for objects:
```suggestion
  { username, password } = req.body;
```